etna/analysis/plotters.py

```diff
@@ -729,3 +733,69 @@ def plot_residuals(
         ax[i].set_title(segment)
         ax[i].tick_params("x", rotation=45)
         ax[i].set_xlabel(feature)
+
+
+TrendTransformType = Union[
+    "ChangePointsTrendTransform", "LinearTrendTransform", "TheilSenTrendTransform", "STLTransform"
+]
+
+
+def __get_labels_names(trend_transform, segments):
```

Comment:
```suggestion
def _get_labels_names(trend_transform, segments):
```

[No double ](https://stackoverflow.com/a/1301369/7415703)